src/azure-cli/service_name.json

```diff
@@ -0,0 +1,688 @@
+[
+  {
+    "Command": "az account",
+    "Description": "Manage Azure subscription information.",
+    "AzureServiceName": "Azure core",
+    "ConversationalServiceName": "Azure core",
+    "URL": "https://docs.microsoft.com/azure/cost-management-billing/manage/create-subscription"
+  },
+  {
+    "Command": "az acr",
+    "Description": "Manage private registries with Azure Container Registries.",
+    "AzureServiceName": "Azure Container Registries",
+    "ConversationalServiceName": "Container Registries",
+    "URL": "https://docs.microsoft.com/azure/container-registry/"
+  },
+  {
+    "Command": "az acs",
+    "Description": "Manage Azure Container Services.",
+    "AzureServiceName": "Azure Container Services",
+    "ConversationalServiceName": "Container Services",
```

Comment:
What is the difference between AzureServiceName and ConversationalServiceName? If I want to aggregate all the commands from the same service in the reference documents, which one should be the key?

Reply:
AzureServiceName has "Azure" while ConversationalServiceName does not based on observation.
@dbradish-microsoft Can you help answer it?